src/main/java/com/dev/cinema/dao/impl/ShoppingCartDaoImpl.java

```diff
@@ -25,6 +27,7 @@ public ShoppingCart add(ShoppingCart shoppingCart) {
             transaction = session.beginTransaction();
             session.save(shoppingCart);
             transaction.commit();
+            log.info("Entity " + shoppingCart + "successful added from the DB");
```

Comment:
```suggestion
            log.info("Entity " + shoppingCart + "successful added to the DB");
```
everywhere
and what about `update()` method?